packages/repository-modernizer/src/create-base-project-structure.js

```diff
@@ -341,5 +385,34 @@ function copyCoreBundlesOrContentPackages(
         }
     });
 }
+/**
+ *
+ * @param String source path of the source project
+ * @param String[] packagingType array for packaging types to match against
+ *
+ * Get list of artifactId of core bundles
+ */
+function getArtifactIdListOfCoreBundle(source, packagingType) {
```

Comment:
Why not modify the existing `copyCoreBundlesOrContentPackages` method to return the list of artifactIds? The internal logic is same. Just that you would only be interested in artifactId list for bundles :
```
            let bundle_artifactId_list = copyCoreBundlesOrContentPackages(
                      project.projectPath,
                      projectPath,
                      constants.BUNDLE_PACKAGING_TYPES,
                      [],
                      conversionStep
                  );
```